libbeat/publisher/queue/diskqueue/docs/on-disk-structures.md

```diff
@@ -0,0 +1,69 @@
+# Disk Queue On Disk Structures
```

Comment:
Thanks for adding this! I am a big fan of documentation/design living next to the source code.